### ISSUE TITLE
Does the training loss of VAE need to include KL loss?

### ISSUE THREAD

**Issue by fyting** [chenguolin/DiffSplat#13]
Does the training loss of VAE need to include KL loss?

**Comment by chenguolin** [comment]
No, we did not apply KL loss and found it to work well already. Please refer to Equ. (4) and Table 4 in the paper.
